db/python/tables/sample.py

```diff
@@ -299,3 +299,23 @@ async def get_sample_with_missing_participants_by_internal_id(
             _query, {'project': project or self.project}
         )
         return {row['id']: row['external_id'] for row in rows}
+
+    async def get_history_of_sample(self, id_: int):
+        """Get all versions (history) of a sample"""
+        keys = [
+            'id',
+            'external_id',
+            'participant_id',
+            'meta',
+            'active+1 as active',
```

Comment:
Why +1?

Reply:
Forces it to an `int`, otherwise mariadb returns like `x01`: https://stackoverflow.com/questions/19256155/python-mysqldb-returning-x01-for-bit-values.

Reply:
Cool, thanks!